LeftPanel.qml

```diff
@@ -395,6 +395,7 @@ Rectangle {
                 onClicked: {
                     parent.previousButton.checked = false
                     parent.previousButton = advancedButton
+                    middlePanel.state = "Advanced"
```

Comment:
Is this correct? Afaik there is no middlePanel called "Advanced"? Anyhow, it doesn't belong in this PR.